WeatherApp/WeatherApp/Feature/CityScreen/CityScreenView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+
+struct CityScreenView: View {
+
+    @ObservedObject var viewModel: CityScreenViewModel
+
+    let columns = [
+        GridItem(.flexible(), spacing: 18),
+        GridItem(.flexible())]
+
+    var body: some View {
+        ZStack {
+            Color.black
+                .ignoresSafeArea()
+
+            if let weather = viewModel.weather {
```

Comment:
If let should come after navbar in a VStack. If you dont have data screen user wont have navigation bar

Make sure to only make parts of the view conditional if they needed it